### DISCUSSION TITLE
Do not use anything from the official Forge latest commits here!!!

### DISCUSSION THREAD

**Discussion by Bullseye110** [Panchovix/stable-diffusion-webui-reForge#91] [General]
Hello!
The updates on the main Forge project (after the 22nd of July, 2024 - a9e0c38 being the latest working commit) broke forge so much that it is legit unusable right now (especially on cloud GPUs and/or with older extensions).
Just do not use anything from these latest commits from the official Forge into this Reforge project, because you risk broking everything with them. Keep the things as they are right now.
Thanks and good luck with this project as Forge is doomed for now!

**Comment by x-ili-x** [comment]
Highres fix, inpainting and img2img for flux are totally broken for me on the main flux fork. I've tried everything to figure it out but it's just screwed. I wish I mirrored it when it worked because my work literally hinges on HRfix/inpainting and img2img and I'm just dead in the water now.
